modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -1117,6 +1117,10 @@ module.exports = (self) => {
           }
         }
       }
+      // PRO-8043: When dealing with ID's only, request based locale inference
+      // is not needed. `false` - default auto locale inference,
+      // `null` - no locale inference.
+      const inferLocale = titlesOrIds.length > 0 ? false : null;
```

Comment:
```suggestion
      const inferLocale = ids.length > 0 ? false : null;
```
If we have some titles, we will not be able to infer the locale or I'm reading this wrong?